### ISSUE TITLE
robots.txt

### ISSUE THREAD

**Issue by dr-norton** [w3c/link-checker#48]
Validator's own robots.txt filters cause errors for link checker itself.
Here's the link: https://validator.w3.org/robots.txt

For example, a webpage with a link like: https://validator.w3.org/checklink?uri=xyz.com

Produces this error: Status: (N/A) Forbidden by robots.txt. The link was not checked due to robots exclusion rules. Check the link manually.

**Comment by dontcallmedom** [comment]
this is on purpose, to avoid creating infinite loops. does it cause particular issues?

**Comment by dr-norton** [comment]
no, actually. it's just more errors on the result page, not a big deal for me, personally. but the whole situation is like a contraption. and sounds like it could be solved easily. say that, checker could check the validator pages as if they had no parameters, or, it could check a special page, or, something else. getting errors and ignoring them... could be the last item on the list.
